packages/zoom-based-markers/package.json

```diff
@@ -19,14 +19,14 @@
     "url": "https://github.com/opentripplanner/otp-ui/issues"
   },
   "dependencies": {
-    "@opentripplanner/core-utils": "^11.4.0",
+    "@opentripplanner/core-utils": "^11.4.1",
```

Comment:
Note that this is to accommodate tests only, and that no release will be triggered for this package.